Tasks/range.js

```diff
@@ -0,0 +1,15 @@
+// Generate int array from given range
+
+;_range = ( ...Range ) => {;
+  let [from,to] = Range
+  if ( to   > from) {;
+    Range = new Array(to - from + 1);
+    for ( let i=from;i<=to;i++ ){
+      Range[i - from] = i;
+       }}else{
+          return [];
+}; return Range;
```

Comment:
```suggestion
  if (to > from) {
    range = new Array(to - from + 1);
    for (let i = from; i <= to; i++) {
      range[i - from] = i;
    }
  } else {
    return [];
  }
  return range;
```
add/remove spaces, add/remove ';', naming